src/features/SocialLinkGroup/types.ts

```diff
@@ -0,0 +1,6 @@
+import { SimpleLineIcons } from '@expo/vector-icons'
+
+export type SimpleIconsLinks = {
+  name: keyof typeof SimpleLineIcons.glyphMap
+  url: string
+}[]
```

Comment:
delete "[]"

Use types like this "SimpleIconsLinks[]" if you need it like array